### PR TITLE
MUSARK-575

### PR DESCRIPTION
**Pull request by rituvesh**
Reduced the distance between buttons.
Tested the GUI for changes all the effected screens.

### CODE REVIEW

**Review by jarlah — CHANGES_REQUESTED** [review]
You must add a unit test

File: src/components/formfields/saveCancel/__tests__/SaveCancel-test.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    const wrapper = shallow(
+      <SaveCancel
+        id="2"
+        saveLabel="Hii"
+        onClickSave={onClickSave}
+        cancelDisabled={true}
+      />
+    );
+
+    expect(wrapper.find('.cancelButton').everyWhere(n => n.hasClass('cancelButton'))).toEqual(true);
```
Comment (jarlah) [inline src/components/formfields/saveCancel/__tests__/SaveCancel-test.js]: Remove all these wrapper.find('.XXXButton').[every|some|has].. they are absolutely usesless.

What i wanted you to do with a class on the button was:

```wrapper.find('.submitButton').simulate('click');```
and then:
```expect(someSubmitButtonSpy.calledOnce).toBe(true)```
thats it.

Other than that, a snapshot test is good enough. 

You dont need wrapper.debug or wrapper.html tests

**Review by jarlah — CHANGES_REQUESTED** [review]
the enzyme test really needs some cleaning up

File: src/components/formfields/saveCancel/__tests__/SaveCancel-test.js (hunk 1 above)
Comment (rituvesh) [reply src/components/formfields/saveCancel/__tests__/SaveCancel-test.js]: Done!

**Review by jarlah — APPROVED** [review]

**Pull request merged**